src/backends/pgfplots.jl

```diff
@@ -175,52 +175,55 @@ end
 
 function _pgfplots_get_plot_kwargs(plt)
     kwargs = KW()
-    kwargs[:style] = ""
+    kwargs[:style] = []
     _pgfplots_get_linestyle!(kwargs, plt)
     _pgfplots_get_marker!(kwargs, plt)
     _pgfplots_get_series_color!(kwargs, plt)
     _pgfplots_get_label!(kwargs, plt)
+    kwargs[:style] = join(kwargs[:style], ',')
     kwargs
 end
 
+
+
 function _pgfplots_axis(plt_series)
     line_type = plt_series[:linetype]
     plt_kwargs = _pgfplots_get_plot_kwargs(plt_series)
-        if line_type == :path
+    if line_type == :path
         PGFPlots.Linear(plt_series[:x], plt_series[:y]; plt_kwargs...)
     elseif line_type == :path3d
         PGFPlots.Linear3(plt_series[:x], plt_series[:y], plt_series[:z]; plt_kwargs...)
     elseif line_type == :scatter
         PGFPlots.Scatter(plt_series[:x], plt_series[:y]; plt_kwargs...)
     elseif line_type == :steppre
-        plt_kwargs[:style] *= "const plot mark right,"
+        plt_kwargs[:style] *= ", const plot mark right"
```

Comment:
... (continued from above) so this line has to use `*=`. If not, I would have to do the "join" in all the `elseif` blocks.